docs/platforms/javascript/common/configuration/integrations/openfeature.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: OpenFeature
+description: "Learn about using Sentry with LaunchDarkly."
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+---
+
+<PlatformContent includePath="feature-flags/prerelease-alert" />
+
+<Alert level="info">
+
+This integration only works inside a browser environment.
+
+</Alert>
+
+_Import name: `Sentry.openFeatureIntegration` and `Sentry.OpenFeatureIntegrationHook`_
+
+The [OpenFeature](https://https://openfeature.dev/) integration tracks feature flag evaluations produced by the OpenFeature SDK. These evaluations are held in memory and, in the event an error occurs, sent to Sentry for review and analysis. **At the moment, we only support boolean flag evaluations.**
+
+```JavaScript
+import * as Sentry from '@sentry/browser';
+import { OpenFeature } from '@openfeature/web-sdk';
+
+Sentry.init({integrations: [Sentry.openFeatureIntegration()]});
```

Comment:
missing `dsn` in the example, with the cool dropdown to pick a project when you're logged in

Reply:
idk how it works... i suspect it's automatic. but it looks like this in the end:

<img width="813" alt="SCR-20241202-iwnq" src="https://github.com/user-attachments/assets/bf040418-6c34-4eb6-889e-9a2898201248">

Reply:
this dropdown is awesome, is it a vscode extension?

Reply:
not a vscode thing, it's on the docs website